lib/commands/index.js

```diff
@@ -26,12 +27,30 @@ module.exports = (robot) => {
   app.post(
     /(?:un)?subscribe/,
     middleware.authenticate,
-    middleware.getResource,
+    middleware.getResource('repo'),
     middleware.getInstallation,
     middleware.pendingCommand.clear,
     subscribe,
   );
 
+  app.post(
+    '/close',
+    middleware.earlyAccess,
+    middleware.authenticate,
+    middleware.getResource(),
+    middleware.getInstallation,
```

Comment:
Do we need `getInstallation`?

Reply:
Yes, I believe the user won't be able to take action without an installation.